README.md

```diff
@@ -18,14 +18,20 @@ package to retrieve (almost) any browser's history on (almost) any platform.
 
 ```python
 from browser_history.browsers import Firefox
+from browser_history.utils import *
 
 f = Firefox.fetch()
 
 his = f.get()
+
+all_browsers = get_history()
+
+all_histories = all_browsers.get()
```

Comment:
This could be named something like `outputs` because it's only the output we're getting and not the actual browser objects.
```suggestion

outputs = get_history()

histories = outputs.get()
```